### DISCUSSION TITLE
IOS:  No Asset Catalog available in new UNO project

### DISCUSSION THREAD

**Discussion by MCAleckson** [unoplatform/uno#13171] [General]
I've successfully rolled out IOS and Android apps for for 3 years with older .Net framework versions and Uno - so I understand the fundamentals of IOS Uno apps into the App Store.  But now, it's a .Net 7.0 project.

So I created a new IOS and Android multiplatform mobile app project using the Uno Solution Template.  I've got all the required provisioning profiles, certificates and such - and the app compiles and runs on both the simulators and physical hardware.

<img width="230" alt="image" src="https://github.com/unoplatform/uno/assets/87875638/a10d6dec-2972-4e80-9efc-e1ec352ba9de">

Now.

I want to send it to TestFlight and then ultimately to the AppStore.  Upon finally getting an .IPA file created (via the Publish command on the .Mobile project head), I use the Transporter app which I've used hundreds of times in the past to attempt to put the app IPA file up in TestFlight on my Apple developer account.

Validation fails with these messages in the Transporter log:

****

Asset validation failed (90046)
Invalid Code Signing Entitlements. Your application bundle's signature contains code signing entitlements that are not supported on iOS. Specifically, value 'development' for key 'aps-environment' in ...

Asset validation failed (90022)
Missing required icon file. The bundle does not contain an app icon for iPhone / iPod Touch of exactly '120x120' pixels, in .png format for iOS versions >= 10.0. To support older versions of iOS, the icon may be required in the bundle outside of an asset catalog. Make sure the Info.plist file includes appropriate entries referencing the file. ...

Asset validation failed (90713)
Missing Info.plist value. A value for the Info.plist key 'CFBundleIconName' is missing in the bundle ...

****

90046 is an easy fix.  However, 90022 and 90713 (dealing with the icons Apple requires) aren't.

Does anyone know where, in new UNO mobile projects targeting both Android and IOS, where and how the AppIconSet is defined, stored and how it is referenced in the project?

Thanks!

**Comment by jeromelaban** [comment]
Thanks for the report! Let's track in https://github.com/unoplatform/uno.templates/issues/229

**Reply by MCAleckson** [reply]
Jerome, thanks for your reply.  I was just coming back into this thread to report that Resizetizer works to create those icons - so I suppose if the officially-supported way to distribute IOS apps is Resizetizer - the the lack of the older-school Asset Catalog is not a problem.

Thanks again.
